### DISCUSSION TITLE
Not getting email notifications

### DISCUSSION THREAD

**Discussion by emcgal** [community/community#66134] [Other Feature Feedback, Questions, & Ideas]
Question

Can someone please tell what I am missing? I don't use Github at the moment but logged some issues and I am not getting any email when I am mentioned, only if the status of the issue is changed.

Settings > Notifications > Participating, @mentions and custom is set to Email and On Github

It is a pain, I have missed comments looking for more feedback. Any ideas why I only get emailed on status change and not mentions?

[Update I'm getting emails from replies here which was one of the reasons I posted. I'm baffled]

**Comment by arafajs** [comment]
Settings > Notifications > Change what you want

**Comment by emcgal** [comment]
@arafajs sorry if I wasn't clear:
Settings > Notifications > Participating, @mentions and custom is  to Email and On Github but I have not been getting any email for mentions on issues reported to the Woocommerce. Only if they change the status to merge or closed but not the mentions.

That's why I posted here. Anyone know why I'm not getting the email?

**Comment by Vijaykv5** [comment]
@emcgal 
Check Your GitHub Notification Settings:

Go to your GitHub account settings by clicking on your profile picture in the top right corner and selecting "Settings."
In the left sidebar, click on "Notifications."
Ensure that your email notification preferences are correctly configured. Specifically, make sure that "Participating" and "@mentions" are set to "Email" and "On GitHub.
Sometimes, GitHub notification emails can end up in your spam or junk folder. Check these folders to see if the emails are being filtered there.

**Reply by emcgal** [reply]
Thanks @Vijaykv5 that's what has me baffled, both email and github are active. I'm getting email notifications from this but not from the Woocommerce posts. I will get a change of status from it but not mentions so I have missed them. It's weird. Nothing in my spam folder either.

Tbh part of the reason I posted here was to see if the email notifications worked and they do here. Completely baffled.

**Comment by emcgal** [comment]
I received an email notification from Vitalii Maletskyi saying they were having the same issue but I don't see the comment. Maybe deleted. 

The problem was never resolved on WordPress threads. Notifications are emailed from this thread no problem but not when I'm mentioned on the WordPress thread which is really annoying as I don't tend to log in regularly here. According to my settings, I should be getting emailed for all mentions. Not sure why it doesn't work on some.
